app/src/main/res/values/strings.xml

```diff
@@ -121,4 +121,7 @@
     <string name="action_agree">AGREE</string>
     <string name="action_dis_agree">DIS AGREE</string>
     <string name="action_add_photo">Add Photo</string>
+    <string name="title_bill_label">Sale amount</string>
+    <string name="title_bill_description">Bill</string>
+    <string name="title_total">Total</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_total` appears to be unused
 Unused resources make applications larger and slow down builds.